services/brig/src/Brig/TURN/API.hs

```diff
@@ -98,6 +108,15 @@ newConfig env = do
         (f:fs) <- shuffleM (toList xs)
         return $ List1.list1 f fs
 
+    limitedList :: List1 TurnURI -> Int -> List1 TurnURI
```

Comment:
How about changing `limitedList` to `limitedList :: List1 TurnURI -> Range 1 10 Int -> List1 TurnURI` and call `fromRange` within the function?

Reply:
sure!